From b08a7f009ddc933da09da1054d88775bfd3ab4ac Mon Sep 17 00:00:00 2001
From: Igor Pecovnik <igor@armbian.com>
Date: Tue, 13 Oct 2026 21:07:18 +0200
Subject: [PATCH 15/15] configs: bananapi-w2: remove boot noise

Remove the debug UART banner. Disable the EFI loader: Armbian boots with boot.scr.

Signed-off-by: Igor Pecovnik <igor@armbian.com>
---
 configs/bananapi-w2_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configs/bananapi-w2_defconfig b/configs/bananapi-w2_defconfig
index 3c3a27f0f5..9b263aafd2 100644
--- a/configs/bananapi-w2_defconfig
+++ b/configs/bananapi-w2_defconfig
@@ -10,12 +10,12 @@ CONFIG_DEBUG_UART_BASE=0x98007800
 CONFIG_DEBUG_UART_CLOCK=27000000
 CONFIG_IDENT_STRING=" BPI-W2"
 CONFIG_DEBUG_UART=y
+# CONFIG_EFI_LOADER is not set
 CONFIG_BOOTSTD_DEFAULTS=y
 CONFIG_CMD_MMC=y
 # CONFIG_NET is not set
 CONFIG_MMC_RTD129X_DW=y
 CONFIG_MMC_RTD129X=y
 CONFIG_DEBUG_UART_SHIFT=2
-CONFIG_DEBUG_UART_ANNOUNCE=y
 CONFIG_DEBUG_UART_SKIP_INIT=y
 CONFIG_SYS_NS16550=y
-- 
2.53.0

